src/cloudmanager-helpers.js

```diff
@@ -296,6 +296,16 @@ function handleError (_error, errorFn) {
   })
 }
 
+function getFormattedFlags (flags) {
+  return Object.keys(flags).filter(flag => flag !== 'programId').map(flag => {
```

Comment:
This is a bit hardcody and is already missing the `imsContextName` flag. See https://github.com/rwalter215/aio-cli-plugin-cloudmanager/pull/1 for an alternate approach which is more dynamic.

Reply:
Your solution looked great. Thanks for taking the time to flesh that out. I merged your changes into my branch and this PR should now be ready to merge